### PR TITLE
Update required ports information

### PR DESCRIPTION
**Pull request by EmilFattakhov**
In preparation for Gemini 3g, the information about the required ports should be updated. 
This PR should not be merged before the release to not create confusion.

### CODE REVIEW

File: docs/protocol/port-forwarding.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ The actual forwarding process will vary based on your router, below is the gener
 2. Advanced Settings > Port Forwarding
 3. Within the port forwarding screen we will see the following fields, all fields have been filled accordingly to our defaults, except for the Computer IP Address, you will replace this with the computer IP address you received in the first steps.
     1. **Computer IP Address:** `192.168.0.25`
-    2. **Protocol:** `TCP`
+    2. **Protocols:** `TCP`, `UDP`
```
Comment (jim-counter) [inline docs/protocol/port-forwarding.md:78]: Not this line, but on line 15 it feels like we should do:

opened up your firewall for TCP traffic on port 30333 with the following command `sudo ufw allow 30333/tcp`
->
opened up your firewall for traffic on port 30333 with the following command `sudo ufw allow 30333/tcp` for TCP and `sudo ufw allow 30333/udp` for UDP

Or maybe just 

opened up your firewall for TCP & UDP traffic on port 30333 with the following command `sudo ufw allow 30333`

**Review by jim-counter — COMMENTED** [review]
One very small change requested. Otherwise looks great thanks Emil!

File: docs/protocol/port-forwarding.md (hunk 1 above)
Comment (EmilFattakhov) [reply docs/protocol/port-forwarding.md:78]: I like the first, more elaborate option. Good point Jim, added 👍

**Comment by EmilFattakhov** [comment]
Thanks, @jim-counter, updated. 
Won't be re-requesting review since it's a minor change :)

File: docs/protocol/port-forwarding.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ keywords:
 ---
 
 :::caution
-If you are using Linux and enabled `ufw` (firewall), make sure you have opened up your firewall for TCP traffic on port 30333 with the following command `sudo ufw allow 30333/tcp`
+If you are using Linux and enabled `ufw` (firewall), make sure you have opened up your firewall for TCP traffic on port 30333 with the following command `sudo ufw allow 30333/tcp` for TSP and `sudo ufw allow 30333/udp` for UDP
```
Comment (jim-counter) [inline docs/protocol/port-forwarding.md:15]:
```suggestion
If you are using Linux and enabled `ufw` (firewall), make sure you have opened up your firewall for traffic on port 30333 with the following commands `sudo ufw allow 30333/tcp` for TCP and `sudo ufw allow 30333/udp` for UDP
```

**Review by jim-counter — CHANGES_REQUESTED** [review]

**Review by jim-counter — APPROVED** [review]

**Review by jim-counter — APPROVED** [review]

**Review by jim-counter — APPROVED** [review]

**Pull request merged**